experimental/ExtendTeamsforM365/client/pages/productDetail.html

```diff
@@ -6,6 +6,24 @@
     <title>Northwind Product</title>
     <link rel="stylesheet" href="/northwind.css" />
     <link rel="icon" href="data:;base64,="> <!-- Suppress favicon error -->
+    <style>
```

Comment:
Move to css file.

Reply:
Changed and shifted to northwind-teams.css